npm/postinstall.js

```diff
@@ -27,6 +27,6 @@ stat("lib", function(error, stats1) {
   installer.on("error", function(err) {
     console.error(`Failed to build ${ pkg.name } automatically. `)
 
-    throw err
+    console.error(err)
```

Comment:
ouch. I forgot to remove this commit why rebasing this branch. I made this change for debugging only.